### PR TITLE
Features/settings page

### PR DESCRIPTION
**Pull request by laurent-mbuyu**
Settings page without header has been set up.
Profile image has been stored locally to prevent any surprise should the link break.
Rendani can use it to add the header component.

You may review it and let me know if there is anything that need attention.
Thanks :)

### CODE REVIEW

**Review by ReinhardtCromhout — APPROVED** [review]

File: src/components/settings.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+// Setting class having the profile details
+
+class Settings extends Component {
+
+    render(){
+
+        return (
+
+            <div>
```
Comment (grootendorste) [inline src/components/settings.js]: O wow, so many spaces

File: src/index.js [hunk 2]
```diff
@@ -1,8 +1,33 @@
...
-registerServiceWorker();
+// import App from './App';
+// import registerServiceWorker from './registerServiceWorker';
+
+// COMPONENTS
+
+import Settings from './components/settings';
+
+class App extends Component {
+
```
Comment (grootendorste) [inline src/index.js]: This space is not needed.

File: src/index.js [hunk 3]
```diff
@@ -1,8 +1,33 @@
...
+import React, { Component } from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import registerServiceWorker from './registerServiceWorker';
+import { BrowserRouter, Route } from 'react-router-dom'
 
-ReactDOM.render(<App />, document.getElementById('root'));
-registerServiceWorker();
+// import App from './App';
```
Comment (grootendorste) [inline src/index.js]: GC can be removed (Ghost Code).

File: src/index.js [hunk 4]
```diff
@@ -1,8 +1,33 @@
...
+            <div> Login Page </div>
+        )
+    }
+
+}
+
+ReactDOM.render(
+    
+    <BrowserRouter>
+        <div>
```
Comment (grootendorste) [inline src/index.js:19]: Is this div needed? I don't see a style applied on it so I'm not sure if this is necessary.

**Review by grootendorste — CHANGES_REQUESTED** [review]
Please look at my comments.

**Pull request merged**